cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/resource/cruisecontrol/PathBuilder.java

```diff
@@ -149,12 +178,30 @@ public PathBuilder withAddBrokerParameters(AddBrokerOptions options) {
     public PathBuilder withRemoveBrokerParameters(RemoveBrokerOptions options) {
         if (options != null) {
             return withAbstractRebalanceParameters(options)
+                    .withParameter(CruiseControlParameters.VERBOSE, String.valueOf(options.isVerbose()))
+                    .withParameter(CruiseControlParameters.SKIP_HARD_GOAL_CHECK, String.valueOf(options.isSkipHardGoalCheck()))
                     .withParameter(CruiseControlParameters.BROKER_ID, options.getBrokers().stream().map(String::valueOf).collect(Collectors.joining(",")));
         } else {
             return this;
         }
     }
 
+    /**
+     * Adds remove disks options to the path
+     *
+     * @param options   Remove-disks options
+     *
+     * @return  Instance of this builder
+     */
+    public PathBuilder withRemoveBrokerDisksParameters(RemoveDisksOptions options) {
+        if (options != null) {
+            return withAbstractRebalanceParameters(options)
```

Comment:
really weird using a method for a rebalance endpoint which is unrelated to the remove disks.

Reply:
I thought that instead of adding extra method which will do similar thing, maybe re-using the existing method would be better. Also I thought by `AbstractREbalanceParamter` we we pointing to something generic that can be used by all the endpoints

Reply:
well no ... if you look at the method it's setting several parameters which make sense for rebalance endpoint only, or?